src/Components/Authorization/Mobile/RegisterForm.tsx

```diff
@@ -40,58 +68,73 @@ export const MobileRegisterForm: FormComponentType = props => {
         </div>
       )}
     </Step>,
-    <Step>
+    <Step validationSchema={Validators.password}>
       {({
         wizard,
         form: { errors, touched, values, handleChange, handleBlur },
       }) => (
         <div>
           <Input
             block
-            error={touched.password && errors.password}
+            error={errors.password}
             name="password"
             placeholder="Password"
             type="password"
             value={values.password}
             onChange={handleChange}
             onBlur={handleBlur}
+            quick
           />
         </div>
       )}
     </Step>,
-    <Step>
+    <Step validationSchema={Validators.name}>
       {({
         wizard,
         form: { errors, touched, values, handleChange, handleBlur },
       }) => (
         <div>
           <Input
             block
-            error={touched.name && errors.name}
+            error={errors.name}
             name="name"
             placeholder="Name"
             type="text"
             value={values.name}
             onChange={handleChange}
             onBlur={handleBlur}
+            quick
           />
         </div>
       )}
     </Step>,
   ]
   return (
-    <Wizard steps={steps} validationSchema={Validators}>
+    <Wizard steps={steps}>
       {context => {
-        const { wizard } = context
+        const { form, wizard } = context
         const { currentStep } = wizard
         return (
           <Container>
+            <ProgressIndicator
+              percentComplete={
+                wizard.isLastStep
+                  ? 1
+                  : (wizard.currentStepIndex + 1) / wizard.steps.length
+              }
+            />
+            <BackButton onClick={wizard.previous as any}>
```

Comment:
@erikdstock We need to figure out the right TS type so we don't need to add `as any` on those. Will look into it